helmfile.d/values/podsecuritypolicies/workload/default.yaml.gotmpl

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.user.constraints }}
 constraints:
 {{- range $key, $value := .Values.user.constraints}}
-  {{ $key }}:{{- unset $value "psaLevel" | toYaml | nindent 4 }}
+  {{ $key }}:{{- unset (deepCopy $value) "psaLevel" | toYaml | nindent 4 }}
```

Comment:
question: does this code change work even when there is no psaLevel set?
i'm thinking since [deepCopy panics](https://helm.sh/docs/chart_template_guide/function_list/#deepcopy-mustdeepcopy) when there is a problem, and i'm wondering if it having no value counts.

Reply:
`deepCopy` just makes a copy of the values, it's `unset` that deals with `psaLevel` and [unset returns without error if the key is not found](https://helm.sh/docs/chart_template_guide/function_list/#unset).

Reply:
Nice, thanks for the explanation!